### PR TITLE
Add Clarity About Downloading and Self-Hosting a Font File

### PR DESCRIPTION
**Pull request by MaedahBatool**
Details about downloading and self-hosting font files were missing from our[ Font Optimization docs](https://nextjs.org/docs/basic-features/font-optimization). This PR addresses these changes.

## Bug

- [x] Related issues linked using #33054

## Documentation / Examples

- [x] Make sure the linting passes by running `yarn lint`

### CODE REVIEW

**Review by molebox — DISMISSED** [review]

File: docs/basic-features/font-optimization.md [hunk 1]
```diff
@@ -78,6 +78,10 @@ Automatic Webfont Optimization currently supports Google Fonts and Typekit with
 
 See [Google Font Display](https://nextjs.org/docs/messages/google-font-display) for more information.
 
+> **Note**: If you want to download and self-host a font file, then our Font Optimization will not work. This is by design!
+>
+> To benefit from font optimization, it's recommended to include the link of the font file in the `Head` component and fetch it from a CDN like Google Fonts.
+
```
Comment (leerob) [inline docs/basic-features/font-optimization.md:84]:
```suggestion
> **Note**: Font Optimization does not currently support self-hosted fonts.
```

Maybe we link to a GitHub discussion here to leave feedback?

Comment (leerob) [reply docs/basic-features/font-optimization.md:84]: It's not _currently_ supported, but we would like to have that eventually.

File: docs/basic-features/font-optimization.md [hunk 2]
```diff
@@ -78,6 +78,7 @@ Automatic Webfont Optimization currently supports Google Fonts and Typekit with
 
 See [Google Font Display](https://nextjs.org/docs/messages/google-font-display) for more information.
 
+> **Note**: Font Optimization does not currently support self-hosted fonts.
```
Comment (styfle) [inline docs/basic-features/font-optimization.md:81]:
```suggestion
> **Note**: Font Optimization does not currently support self-hosted fonts.

```

**Review by styfle — APPROVED** [review]
Thanks!

**Pull request merged**